Assets/Scripts/UI/DraggableObject.cs

```diff
@@ -45,6 +48,9 @@ public virtual void OnBeginDrag(PointerEventData eventData)
             isDragging = true;
             RectTransformUtility.ScreenPointToLocalPointInRectangle(rectTransform, eventData.position, eventData.pressEventCamera, out _pointerOffset);
 
+            //Return the draggable block to normal size once it is being dragged
+            transform.localScale = new Vector3(1.0f, 1.0f, 1.0f);
```

Comment:
Let's add a "dragging scale" Vector3 field (as I mentioned above) and use that instead of hardcoding this Vector3.